src/types.ts

```diff
@@ -45,6 +45,14 @@ export enum Currency {
     EUR = "eur"
 }
 
+export enum TradingView {
+    ADVANCE = 'advance',
```

Comment:
Can we change this to `ADVANCED`? It is listed that way on the site

Reply:
copy